packages/controllers/profile_detail.coffee

```diff
@@ -0,0 +1,11 @@
+if Meteor.isClient
+  Template.profileDetail.onCreated ->
+    @subscribe('userProfileDetail', this.data.profileId)
+  
+  Template.profileDetail.helpers
+    userProfile: ->
+      UserProfiles.findOne()
```

Comment:
It would be safer to use the ids in the finds, even if there's no legit way to subscribe to more than one - we'll probably have to add it later anyway.